docs/examples/compute/instance/env-vars

```diff
@@ -11,7 +11,7 @@ export TF_VAR_region="<region in which to operate, example: us-ashburn-1, us-pho
 export TF_VAR_compartment_ocid="<compartment OCID>"
 
 ### Public/private keys used on the instance
-export TF_VAR_ssh_public_key_path=<path to public key>
-export TF_VAR_ssh_private_key_path=<path to private key>
+export TF_VAR_ssh_public_key=<path to public key>
```

Comment:
"<path to public key>" should also be changed to "<public key>"
same for the private key

Reply:
It seems we have a change in progress that will take care of this. It will be out soon.